### DISCUSSION TITLE
How to delete languages in details?

### DISCUSSION THREAD

**Discussion by vanaur** [community/community#22935] [New to GitHub]
<p>Hello,<br>
I inserted a new folder, including a lot of files, in my repository. Initially, the language detected in the “language details” was C++, because my project is in C++. The new folder only contains assembler files, but the language details display more than expected, and what’s more, incorrect: </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/5de2f92199b40a8815f8acb23bd8a34b3a981f03.png" data-download-href="/uploads/short-url/doyGW3K08FX0VFI7mUxKVASqE8z.png?dl=1" title="Capture.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/28791317/181073752-f20be082-8962-48d4-aaf2-385a7518a115.png" alt="Capture.PNG" data-base62-sha1="doyGW3K08FX0VFI7mUxKVASqE8z" width="690" height="38" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/5de2f92199b40a8815f8acb23bd8a34b3a981f03_2_690x38.png, https://aws1.discourse-cdn.com/github/optimized/1X/5de2f92199b40a8815f8acb23bd8a34b3a981f03_2_1035x57.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/5de2f92199b40a8815f8acb23bd8a34b3a981f03.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/5de2f92199b40a8815f8acb23bd8a34b3a981f03_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Capture.PNG</span><span class="informations">1228×69 6.97 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Normally, there should only be C++ and Assembly.</p>
<p>How can I delete the others? This gives incorrect information about the project.</p>
<p>Thank you by advance!</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/vanaur">@vanaur</a>,</p>
<p>GitHub uses Linguist to source these stats. If it is reporting the wrong language for your repository, I recommend that you use <a href="https://github.com/github/linguist#troubleshooting" rel="nofollow noopener">these troubleshooting steps</a> to figure out why and take action as necessary.</p>
<p>Thanks!</p>

**Comment by vanaur** [comment]
<p>Okay, thanks for your help, I’ll see what I can do :smiley:</p>
